process-services/latest/using/workflows.md

```diff
@@ -0,0 +1,246 @@
+
+A workflow is a process that controls a specific task, such as a document review. Each task can be assigned to one or more people when a workflow is being created. Workflow creators can select to automatically send a notification email to users that are assigned a task as part of the workflow, but whether an email is sent or not the task will be visible in the users My Tasks.
+
+Once all the task actions required by a workflow have been finished, the workflow will move from active to completed status. At this point you’re free to delete the workflow.
+Individual users manage their own tasks, and the person who created a workflow manages the workflow.
+
+## Creating a tenant
+
+You need to create a tenant before you can start creating workflows. 
+
+To create a tenant, complete the following steps.
+
+1. Log in as admin to Alfresco Process Services as the admin user.
+
+2. Select **Identity Management**.
+
+3. Select **Tenants**.
+
+4. Select **Create tenant**.
+
+      The **Create tenant** dialog box appears.
+
+5. Enter the following details as per your requirements.
+
+      * Enter the **Name** – The name of your tenant, for example, xxx.
+      * Specify the **Maximum allowed users in this tenant** - This is set at 10 bBy default, it is set at 10.
+
+6. Select the **Deploy preconfigured workflows** checkbox.
+
+7. Add the details about the tenant owner including **First name**, **Last name**, **Email**, and **Password**. 
+
+      You can also assign a tenant owner from the list of created users.
+
+8. Click **Save**.
+
+## Configuring a preconfigured workflow
+
+All users in a tenant can access the preconfigured workflows. The owner of the tenant is the only user that can make edits to the workflow process.
+
+To configure a preconfigured workflow, complete the following steps.
+
+1. Log in to Alfresco Process Services.
+
+2. Select **App Designer**.
+
+3. Select the preconfigured workflow that you want to change.
+
+4. Make the edits and save them.
+
+      The edited changes are now visible to all the users of the tenant.
+
+## Creating a user
+
+To create a user in a tenant, complete the following steps.
+
+1. Log in to Alfresco Process Services.
+
+2. Select **Identity Management**.
+
+3. Select **User**.
+
+4. Select **Create user**.
+
+      The **Create user** window appears.
+
+5. Enter the following details.
+
+      * **Email** of the user.
+	  * **Password** of the user.
+      *	**First name** of the user.
+      * **Last name** of the user.
+	  * Select your **Tenant** from the drop-down list.
+	  * Enter the **Company** name.
+      * Select the **Status** and **Type**.
+
+6. Click **Save**.
+
+## Creating a group
+
+To create a group in a tenant, complete the following steps.
+
+1. Log in to Alfresco Process Services.
+
+2. Select **Identity Management**.
+
+3. Select **Organization**.
+
+4. Select your tenant from the drop-down list.
+
+5. Select **Create group**. Name your group and click **Save**.

```

Comment:
after selecting your tenant a "Workflow group" will be visible be default to make the other users to view the worklfow we need to add those users in this group